certbot/tests/main_test.py

```diff
@@ -1164,6 +1165,69 @@ def test_update_registration_with_email(self, mock_utility, mock_email):
                             email in mock_utility().add_message.call_args[0][0])
 
 
+class UnregisterTest(unittest.TestCase):
+    def setUp(self):
+        self.patchers = {
```

Comment:
In the future, we may want to avoid using `mock.patch` like this. See the comments [here](https://github.com/certbot/certbot/pull/3784#issuecomment-262337637). I don't think this has to change though. We do it in plenty of other places.